keyboards/ploopyco/ploopyco.h

```diff
@@ -37,3 +37,5 @@ enum ploopy_keycodes {
 
 bool encoder_update_user(uint8_t index, bool clockwise);
 bool encoder_update_kb(uint8_t index, bool clockwise);
+void toggle_drag_scroll(void);
+void cycle_dpi(void);
```

Comment:
```suggestion
void cycle_dpi(void);
```
Keep newline at EOF.

Reply:
Stared at this way too long before I saw the "keep newline at EOF". done! Thank you for reviewing